src/app/admin/subject/[slug]/[unit]/[articleNumber]/page.tsx

```diff
@@ -0,0 +1,66 @@
+"use client";
+import ArticleCreator from "@/app/article-creator/_components/ArticleCreator";
+import { getUser } from "@/components/hooks/users";
+import Footer from "@/components/ui/footer";
+import Navbar from "@/components/ui/navbar";
+import { User } from "@/types/user";
+import { useEffect, useState } from "react";
+
+const Page = ({ params }: { params: { slug: string } }) => {
+  const [loading, setLoading] = useState<boolean>(true);
+  const [error, setError] = useState<string | null>(null);
+  const [user, setUser] = useState<User | null>(null);
+
+  const pathParts = window.location.pathname.split("/").slice(-2);
+  const formattedTitle = `Unit ${pathParts[1]?.charAt(0).toUpperCase() + pathParts[1]!.slice(1)}: ${pathParts[0]?.charAt(0).toUpperCase() + pathParts[0]!.slice(1)}`.replace(/-/g, " ");  
```

Comment:
???

Reply:
I think this is fine, we have some guarantees about subject names and unit names